### PR TITLE
Allows for SELECT DISTINCT in queries

### PR DESCRIPTION
**Pull request by luketlancaster**
Thought I'd give this a shot. There's a _real_ good chance I haven't though through this 100%, but figured why not?

### CODE REVIEW

**Comment by shadowhand** [comment]
Awesome! Can you update the docs (README) too?

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ echo $select->sql();
 - `orderBy(array ...pairs)` either `[column]` or `[column, direction]`
 - `limit(int limit)`
 - `offset(int offset)`
+- `distinct(bool or null)` defaults to `true`
```
Comment (luketlancaster) [inline README.md]: watcha think about that, @shadowhand?

File: README.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ echo $select->sql();
 - `orderBy(array ...pairs)` either `[column]` or `[column, direction]`
 - `limit(int limit)`
 - `offset(int offset)`
+- `distinct(bool or null)` defaults to `false`
```
Comment (luketlancaster) [inline README.md]: Fixed...defaults to *false*. Should really have my coffee before I try to code...

**Comment by shadowhand** [comment]
Released in 1.3.0 and 0.3.1. Thanks!

**Comment by luketlancaster** [comment]
👏🏻👍🏻👏🏻

**Pull request merged**